test/integration/singlecluster/controller/dra/suite_test.go

```diff
@@ -144,19 +144,7 @@ func managerSetup(modifyConfig func(*config.Configuration)) framework.ManagerSet
 
 // Helper function to create a ResourceClaim
 func makeResourceClaim(name, namespace, deviceClassName string, count int64) *resourcev1.ResourceClaim {
```

Comment:
is this utility function of any value? can we instead use the builder and remove this?